examples/motion/main.go

```diff
@@ -0,0 +1,37 @@
+// Package main demonstrates how to move the robot through ALMotion
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"bitbucket.org/swoldt/pkg/xerrors/iferr"
```

Comment:
people reading examples might not be too familiar with Go:
please keep the example dead simple without unnecessary dependency.

also, I am not sure why, iferr.Exit prints errors like:
```
ERRO[2019-11-29 23:10:30] call walkTo failed: ISerialization error Status Read Past End  caller="main.go:36" fName=main.main
```
while the same error with panic prints:
```
panic: call walkTo failed: ALMotionwalkTo not possible. This robot has no legs.

goroutine 1 [running]:
main.main()
        /home/ludo/go/src/github.com/lugu/qiloop/examples/motion/main.go:42 +0x27a
exit status 2
```